README.md

```diff
@@ -42,7 +42,8 @@ or microbiome data. Alpha diversity describes an estimate of the total number
 of species in a sample. Beta diversity describes the differences between
 samples. Below are some example of then number of metrics you can use.
 
-![Plot of seven different alpha diversity metrics](images/alpha_diversities.png)
+<img src="images/alpha_diversities.png" alt="Drawing" style="width: 850px;"/>
```

Comment:
Change the width to 100% like this

```html
<img src="images/alpha_diversities.png" alt="Drawing" style="width: 100%;"/>
```

See https://stackoverflow.com/a/15458733/6873133